src/xcode/ENA/ENA/Source/AppDelegate & Globals/ENATaskExecutionDelegate.swift

```diff
@@ -113,6 +113,15 @@ class TaskExecutionHandler: ENATaskExecutionDelegate {
 							}
 						}
 						
+						group.enter()
+						DispatchQueue.global().async {
+							Log.info("Check for invalid certificates", log: .background)
+							self.checkCertificateValidityStates {
+								group.leave()
+								Log.info("Done checking for invalid certificates.", log: .background)
+							}
+						}
```

Comment:
As these are all async tasks, the order in which they are triggered will have no impact on the priority. They are all started in parallel.